update.sh

```diff
@@ -8,44 +8,52 @@
-  : > "$HOME/.rhino/updates/configuration"
+if [[ ! -f "/usr/share/rhino/updates/configuration" ]]; then
+  mkdir -p /usr/share/rhino/{config,updates}
+  echo "alias rhino-config='mkdir /usr/share/rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config /usr/share/rhino/config/config-script/ && python3 /usr/share/rhino/config/config-script/config.py && rm -rf /usr/share/rhino/config/config-script'" >> ~/.bashrc
+  : > "/usr/share/rhino/updates/configuration"
 fi
 
 # Check to see whether the rhino-config v2 update has been applied, which converts Rhino into a command-line utility.
-if [[ ! -f "$HOME/.rhino/updates/config-v2" ]]; then
-  mkdir ~/rhinoupdate/distro
-  git clone https://github.com/rollingrhinoremix/distro ~/rhinoupdate/distro
-  mv ~/rhinoupdate/distro/.{bashrc,bash_aliases} ~
-  : > "$HOME/.rhino/updates/config-v2"
+if [[ ! -f "/usr/share/rhino/updates/config-v2" ]]; then
+  mkdir /usr/share/rhino/rhinoupdate/distro
+  git clone https://github.com/rollingrhinoremix/distro /usr/share/rhino/rhinoupdate/distro
+  mv /usr/share/rhino/rhinoupdate/distro/.{bashrc,bash_aliases} ~
+  : > "/usr/share/rhino/updates/config-v2"
 fi
 
 # Check to see whether Nala is installed.
-if [[ ! -f "$HOME/.rhino/updates/nala" ]]; then
+if [[ ! -f "/usr/share/rhino/updates/nala" ]]; then
   sudo apt install nala -y
-  : > "$HOME/.rhino/updates/nala"
+  : > "/usr/share/rhino/updates/nala"
 fi
 
 # Install latest rhino-config utility
-mkdir ~/rhino-config
-cd ~/rhino-config
+mkdir /usr/share/rhino/rhino-config
+cd /usr/share/rhino/rhino-config
 wget -q --show-progress --progress=bar:force https://github.com/rollingrhinoremix/rhino-config/releases/latest/download/rhino-config
 chmod +x rhino-config
 sudo mv rhino-config /usr/bin
-rm -rf ~/rhino-config
+rm -rf /usr/share/rhino/rhino-config
 
 # Install the latest rhino-deinst utility
-mkdir ~/rhinoupdate/rhino-deinst
-cd ~/rhinoupdate/rhino-deinst
+mkdir /usr/share/rhino/rhinoupdate/rhino-deinst
+cd /usr/share/rhino/rhinoupdate/rhino-deinst
 wget -q --show-progress --progress=bar:force https://github.com/rollingrhinoremix/rhino-deinst/releases/latest/download/rhino-deinst
 chmod +x rhino-deinst
 sudo mv rhino-deinst /usr/bin
 
 # Automatically install the latest Linux kernel onto the system if it has not been installed already.
-if [[ ! -f "$HOME/.rhino/config/5-18-11" ]]; then
+if [[ ! -f "/usr/share/rhino/config/5-18-11" ]]; then
     cd ~/rhinoupdate/kernel/
     wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.18.11/amd64/CHECKSUMS &
     wget -q --show-progress --progress=bar:force https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.18.11/amd64/linux-headers-5.18.11-051811-generic_5.18.11-051811.202207121541_amd64.deb &
```

Comment:
Please update to the last version of the script, which is utilising the 5.19 kernel.

Reply:
Unless you plan on merging soon, I think this could be in another PR.